v2/src/ctrlfbe/views.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+
+from ctrlfbe.models import Note
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class NoteAPIView(APIView):
+    authentication_classes: List[str] = []
+
+    def get(self, request):
+        current_cursor = int(request.query_params["cursor"])
+        notes = Note.objects.all()[current_cursor : current_cursor + 30]
```

Comment:
```suggestion

        # constans.py
        MAX_PRINTABLE__NOTE_COUNT = 30
        
        ...
        
        # views.py
        ...
        notes = Note.objects.all()[current_cursor : current_cursor + MAX_PRINTABLE__NOTE_COUNT]
```

매직넘버는 상수로 뺄 수 있을 것 같음~

Reply:
764f33c 반영했습니다.

Reply:
앗.. MAX_PRINTABLE__NOTE_COUNT 에서 _ 2개 들어간건.. 오타임 ㅠ

`MAX_PRINTABLE_NOTE_COUNT` 로.. 내가 고쳐서 push 해둘게